modules/remotecontrol/RemoteControl.js

```diff
@@ -45,6 +74,9 @@ class RemoteControl {
         this._initialized = true;
```

Comment:
I'm not sure I'm understanding the question. Are you talking about setting the event handler or something else? There shouldn't be any active events before the init...